.github/workflows/test.yml

```diff
@@ -1,9 +1,7 @@
 name: Test
 
-on:
-  push:
-    branches:
-    - main
+on: [push]
```

Comment:
Take a look at https://github.com/vega/vega-embed/blob/673b6f03e49e8d58c4478d4b91a5833e0048aeec/.github/workflows/test.yml#L3. We use a different format to support tests on prs from internal and external.